src/charm.py

```diff
@@ -210,6 +211,14 @@ def _bootstrap_mysqlrouter(self) -> bool:
 
             self.unit_peer_data[UNIT_BOOTSTRAPPED] = "true"
 
+            try:
+                subprocess.check_call(["open-port", "6446/tcp"])
+                subprocess.check_call(["open-port", "6447/tcp"])
+                subprocess.check_call(["open-port", "6448/tcp"])
+                subprocess.check_call(["open-port", "6449/tcp"])
+            except subprocess.CalledProcessError:
+                logger.exception("failed to open port")
```

Comment:
I would prefer that we iterate ports, for better error logging

```suggestion
            for port in ["6446", "6447", "6448", "6449":
                try:
                    subprocess.check_call(["open-port", f"{port}/tcp"])
                except subprocess.CalledProcessError:
                    logger.exception(f"failed to open port {port}")
```